src/samtools/samtools_fasta/config.vsh.yaml

```
@@ -0,0 +1,189 @@
+        description: |
+          Only output alignments with any bits set in INT present in the FLAG field. INT can be specified 
+          in hex by beginning with `0x' (i.e. /^0x[0-9A-F]+/), in octal by beginning with `0' 
+          (i.e. /^0[0-7]+/), as a decimal number not beginning with '0' or as a comma-separated list of 
+          flag names. [0]
+        example: 0
+      - name: --excl_flags_all
+        alternatives: -G
+        type: integer
+        description: |
+          Only EXCLUDE reads with all of the bits set in INT present in the FLAG field. INT can be specified 
+          in hex by beginning with `0x' (i.e. /^0x[0-9A-F]+/) or in octal by beginning with `0' 
+          (i.e. /^0[0-7]+/). [0]
+        example: 0
+      - name: --aux_tag
+        alternatives: -d
+        type: string
+        description: |
+          Only output alignments containing an auxiliary tag matching both TAG and VAL. If VAL is omitted 
+          then any value is accepted. The tag types supported are i, f, Z, A and H. "B" arrays are not 
+          supported. This is comparable to the method used in samtools view --tag. The option may be specified 
+          multiple times and is equivalent to using the --aux_tag_file option.
+      - name: --aux_tag_file
+        alternatives: -D
+        type: string
+        description: |
+          Only output alignments containing an auxiliary tag matching TAG and having a value listed in FILE. 
+          The format of the file is one line per value. This is equivalent to specifying --aux_tag multiple times.
+      - name: --casava
+        alternatives: -i
+        type: boolean_true
+        description: add Illumina Casava 1.8 format entry to header (eg 1:N:0:ATCACG)
+      - name: --compression
+        alternatives: -c
+        type: integer
+        description: set compression level when writing gz or bgzf fasta files.
+        example: 0
+      - name: --index1
+        alternatives: --i1
+        type: file
+        description: write first index reads to FILE.
+      - name: --index2
+        alternatives: --i2
+        type: file
+        description: write second index reads to FILE.
+      - name: --barcode_tag
+        type: string
+        description: Auxiliary tag to find index reads in. [BC]
+        example: "BC"
+      - name: --quality_tag
+        type: string
+        description: Auxiliary tag to find index quality in. [QT]
+        example: "QT"
+      - name: --index_format
+        type: string
+        description: |
+          string to describe how to parse the barcode and quality tags. For example:
+          [i14i8]: the first 14 characters are index 1, the next 8 characters are index 2.
+          [n8i14]: ignore the first 8 characters, and use the next 14 characters for index 1.
+          If the tag contains a separator, then the numeric part can be replaced with '*' to mean 
+          'read until the separator or end of tag', for example: [n*i*].
```

```suggestion
          string to describe how to parse the barcode and quality tags. For example:
          
          * `i14i8`: the first 14 characters are index 1, the next 8 characters are index 2.
          * `n8i14`: ignore the first 8 characters, and use the next 14 characters for index 1.
            If the tag contains a separator, then the numeric part can be replaced with '*' to mean 
            'read until the separator or end of tag', for example: `n*i*`.
```

Could we update this to markdown formatting?